### PR TITLE
Vendor update: helmclient

### PR DESCRIPTION
**Pull request by tomahawk28**
toward: https://github.com/giantswarm/giantswarm/issues/6157

### CODE REVIEW

**Review by oponder — APPROVED** [review]
FYI: When it's just vendor updates, no need for review or approval.

**Comment by rossf7** [comment]
Let's keep this on hold until we're ready to roll out the tiller upgrade.

@tomahawk28 This is a vendor PR so it doesn't need approval. But in this case, I think the `helmclient` PR should have been kept open until the helm upgrade was fully tested.

**Comment by rossf7** [comment]
Closing in favor of #87

**Pull request closed without merging**
